### PR TITLE
remove obsolete twilio migrations

### PR DESCRIPTION
**Pull request by nickpell**
The only active ```SmsGatewayFeeCriteria``` for Twilio will be those created in ```add_twilio_gateway_fee_for_migration```.  Existing environments will have the obsolete fee criteria deactivated, future environments just won't have the obsolete ones at all.  These fees can be deactivated because we now pull the cost per message from the Twilio API.

The motivation for doing this now is to remove a usage of ```csv``` that is incompatible with python 3.

@dimagi/py3

### CODE REVIEW

File: corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    to_deactivate = SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        country_code__isnull=False,
+        is_active=True,
+    )
+    remaining = SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+    ).exclude(
+        id__in=to_deactivate.values('id')
```
Comment (gcapalbo) [inline corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py]: should this be `values_list('id', flat=True)`?

File: corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    assert SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        is_active=True,
+    ).count() == 2
+    assert SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        country_code__isnull=True,
+        is_active=True,
+    ).count() == 2
```
Comment (gcapalbo) [inline corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py]: how come these two are still needed if the smsbillables look up the fees afterwards?

File: corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py (hunk 1 above)
Comment (millerdev) [reply corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py]: I think `values_list('id', flat=True)` will result in two queries, the first to get the ids and put them in a list (in Python), and the second like
```sql
SELECT ... WHERE id IN ?
```
`?` is replaced with the list of ids.

On the other hand `values('id')` will make a single query like
```sql
SELECT ... WHERE id IN (SELECT id FROM ...)
```
the second select statement is the SQL of the `values('id')` queryset.

Comment (gcapalbo) [reply corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py]: i see, thanks for the explanation

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**